Bottom: 81045ba10948fa56b19a7ffbe70da7984caf87df
Top:    76fb8677ced4c27de6edc97d2d5a512988dbad7d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-15 18:58:48 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
